ex4/ex4.c

```diff
@@ -10,7 +10,24 @@
 
 int main(void)
 {
-    // Your code here    
+    // Your code here
+    int rc = fork();
+    if (rc == 0) {
+        printf("Child running.. \n");
+        char *myargs[2];
+        myargs[0] = strdup("/bin/ls");
```

Comment:
I am getting implicit declaration warnings and I think it's because strdup is not in the standard library - you have to #include string.h. When I run this I can see the child running but it does not execute an ls - I wonder if this will be fixed if you import the string library?

Reply:
I was wondering why I was getting a warning, this fixed it! As for ls, it was always working for me, but maybe it should now be completely fixed since the string library was included